src/RadioFrequency.hpp

```diff
@@ -81,12 +84,28 @@ class RadioFrequency {
     return BASE_KHZ + value;
   }
 
+  /**
+   * VHF Voice channels range from 118000 kHz up to not including 137000 kHz
+   * Valid 8.33 kHz channels must be a multiple of 5 kHz
+   * Due to rounding from 8.33 kHz to multiples of 5 (for displaying), some
+   * channels are invalid. These are matched by (value % 25) == 20.
+   */
   void SetKiloHertz(unsigned khz) {
-    value = khz >= BASE_KHZ && khz < MAX_KHZ
+    value = (khz >= MIN_KHZ && khz < MAX_KHZ) &&
+            (khz % 5 == 0) &&
+            (khz % 25 != 20)
       ? (khz - BASE_KHZ)
       : 0;
   }
 
+  void OffsetKiloHertz(int khz_offset) {
+    auto new_khz = GetKiloHertz() + khz_offset;
+    if ((new_khz % 25) == 20) {
+      new_khz += std::copysign(5, khz_offset);
```

Comment:
This switches to floating point math.

Reply:
Replaced std::copysign by a conditional operator.